Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-watch.txt

MongoDB\\Collection::watch()
============================

.. meta::

>    :description: Execute a change stream operation on a collection using `MongoDB\Collection::watch()` to monitor collection-level changes.

Reviewer: ```suggestion
   :description: Execute a change stream operation on a collection to monitor collection-level changes with the MongoDB PHP Library.
```
